docs/source/community/community-call-notes/2021-april.md

```diff
@@ -33,11 +33,11 @@ For more discussion on the format of these calls, see the thread [here](https://
 - **Gonzalo** Internationalization workflow for extension developers
 
     - Some feedback... :man-facepalming: 
-        - ![Text asking why there is a delay for publishing the language packs needed for internationalization](https://i.imgur.com/mho6Ka9.png)
+        - ![Text asking why there is a delay for publishing the language packs needed for internationalization ](https://i.imgur.com/mho6Ka9.png)
```

Comment:
I'm working on this

Reply:
```suggestion
        - ![A comment from a community members on internationalization: As I see, there is a delay with the creation and publishing of language packs. We use Jupyter Lab as a base for one of our products and we are interested in using language lacks in it (especially Chinese and Japanese. What is the reason for the delay and is it possible to help the project somehow? We are interested in contributing to the Jupyter Lab project in many domains, for example, sofrware development, testing and translation.](https://i.imgur.com/mho6Ka9.png)
```

Reply:
Per Ely's suggestion, we could just delete the image and replace with this text. The image doesn't serve information in a better way than plain text. There was discussion earlier of hesitancy to change past records (archival mindset). However, I feel that making documentation accessible for future use is a higher priority than maintaining an inaccessible record of the past.

Reply:
In our alt-text writing guidelines (such as the upcoming SPEC document), we have a 'best practices' section. We could make an example similar to those to show comparison of what not to do.